src/functions.php

```diff
@@ -331,3 +331,14 @@ function json_encode($value, $options = 0, $depth = 512)
 
     return $json;
 }
+
+/**
+ * Wrapper for the hrtime() or microtime() functions
+ * (depending on the PHP version, one of the two is used)
+ *
+ * @return float|mixed UNIX timestamp
+ */
+function current_time()
+{
+    return function_exists('hrtime') ? hrtime(true) / 1e9 : microtime(true);
```

Comment:
Should be `1e7`, right?

https://3v4l.org/bZ1uo

Reply:
Right, my mistake. Fixed.